.github/workflows/core-test.yml

```
@@ -17,10 +17,10 @@ jobs:
         uses: actions/checkout@v2
         with:
           path: ./cbioportal
-      - name: 'Set up JDK 11'
+      - name: 'Set up JDK 17'
         uses: actions/setup-java@v1
         with:
-          java-version: 11
+          java-version: `17`
```

minor but looks like we have the version without quotes in other workflows.
```suggestion
          java-version: 17
```